avm/res/cache/redis-enterprise/database/access-policy-assignment/main.bicep

```diff
@@ -0,0 +1,52 @@
+metadata name = 'Azure Managed Redis (Preview) Database Access Policy Assignment'
+metadata description = 'This module deploys an access policy assignment for an Azure Managed Redis (Preview) database.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Optional. Name of the access policy assignment.')
+param name string?
+
+@description('Required. Object ID to which the access policy will be assigned.')
+param objectId string
```

Comment:
May make sense to rename to `userObjectId` to be more description. What do you think?

Reply:
Sure, I renamed it